<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QComboBox" name="month_input">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>69</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
   <item>
    <property name="text">
     <string>Jan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Feb</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Apr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>May</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jun</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jul</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Aug</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sep</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Oct</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nov</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dec</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="year_input">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
   <item>
    <property name="text">
     <string>2020</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2022</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2023</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2024</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2025</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2026</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2026</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2028</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2029</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2030</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="date_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="main_bank_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
   <item>
    <property name="text">
     <string>BDO Unibank Inc.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bank of the Philippine Islands (BPI)</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="first_name_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="address_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="city_input">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
   <item>
    <property name="text">
     <string>Manila</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Valenzuela</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Caloocan</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>142</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Main Bank</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bank_4_input">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Bank#</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Bank Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="bank_type_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
   <item>
    <property name="text">
     <string>Debit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Credit</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="branch_input">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
   <item>
    <property name="text">
     <string>Hermosa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sagandaan</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 16pt &quot;MS Shell Dlg 2&quot;;

</string>
   </property>
   <property name="text">
    <string>Branch</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bank_3_input">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bank_2_input">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bank_1_input">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="last_name_input">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_bt">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 26pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
   <property name="text">
    <string>             Bank Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="validator_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 255);
font: 26pt &quot;MS Shell Dlg 2&quot;;
color:white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
